tutor/commands/compose.py

```diff
@@ -60,23 +93,87 @@ def job_runner(self, config: Config) -> ComposeJobRunner:
         raise NotImplementedError
 
 
+class MountParam(click.ParamType):
+    """
+    Parser for --mount arguments of the form "service1[,service2,...]:/host/path:/container/path".
+    """
+
+    name = "mount"
+    MountType = t.Tuple[str, str, str]
```

Comment:
What about just `Mount`? I would expect a type called `MountType` to mean something like an enum of the different types of mounts.

Reply:
Oops I missed that comment, sorry about that...

I picked the "Type" suffix because it's not a class of which we want to create instances. But I'm not bent on it.

Reply:
No worries. I'm not bent on it either.